src/Aspire.AppHost.Sdk/SDK/Sdk.in.targets

```diff
@@ -91,52 +93,19 @@
     </PropertyGroup>
 
     <!-- At this point, we should have the version either by CPM or PackageReference, so we fail if not. -->
-    <Error Condition="'$(_AppHostVersion)' == ''" 
+    <Error Condition="'$(_AppHostVersion)' == ''"
        Text="$(MSBuildProjectName) is a .NET Aspire AppHost project that needs a package reference to Aspire.Hosting.AppHost version 8.2.0 or above to work correctly." />
 
     <!--The following is the list of the RIDs that we currently multitarget for DCP and Dashboard packages-->
     <ItemGroup>
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x86" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-arm64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="linux-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="linux-arm64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="osx-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="osx-arm64" />
+      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x64;win-x86;win-arm64;linux-x64;linux-arm64;osx-x64;osx-arm64" />
     </ItemGroup>
 
-    <!-- If running in .NET Core, DOTNET_HOST_PATH is set to point to the dotnet executable being used
-    for the build. -->
-    <PropertyGroup>
-      <_DotNetHostPath>$(DOTNET_HOST_PATH)</_DotNetHostPath>
-    </PropertyGroup>
-
-    <!-- If running on .NET Framework MSBuild, then DOTNET_HOST_PATH won't be set, so we need to construct
-    the path using well-known properties. -->
-    <PropertyGroup Condition="'$(_DotNetHostPath)' == ''">
-      <_DotNetHostDirectory>$(NetCoreRoot)</_DotNetHostDirectory>
-      <_DotNetHostFileName>dotnet</_DotNetHostFileName>
-      <_DotNetHostFileName Condition="'$(OS)' == 'Windows_NT'">dotnet.exe</_DotNetHostFileName>
-
-      <_DotNetHostPath>$(_DotNetHostDirectory)\$(_DotNetHostFileName)</_DotNetHostPath>
-    </PropertyGroup>
```

Comment:
none of this is necessary if we're not spawning binaries anymore in the SDK-delivered targetst